docs/index.md

```diff
@@ -0,0 +1,301 @@
+# Granite Framework
+
+Granite is a business actions architecture for Rails apps.
```

Comment:
Let's try to be more formal: how about saying 'applications' instead of 'apps'?

Reply:
sure thing 👍